README.md

```diff
@@ -3,37 +3,102 @@
 
 ## Introduction
 
-The is an Ansible playbook for Islandora CLAW. It also has a vagrant file to bring up a development
-environment virtual machine for Islandora CLAW.
+This is the development environment virtual machine (vm) for running Islandora 7.x-2.x (a.k.a. [Islandora CLAW](https://github.com/Islandora-CLAW/CLAW)). 
 
-This virtual machine **should not** be used in production **yet**.
+_This virtual machine **should not** be used in production **yet**._
+
+* It should work on any operating system that supports Ansible, Git, VirtualBox and Vagrant. 
+
+* It uses a suite of [Ansible](https://www.ansible.com/how-ansible-works) playbooks called within the Vagrantfile's provisioners section to install [Islandora CLAW](https://github.com/Islandora-CLAW/CLAW), its dependencies, additional software, user roles and more.
+
+* The information listed below is to serve as a quick getting started guide. 
+  * For more detailed information please navigate to the Technical documentation [here](/docs/index.md)
+
+---
 
 ## Requirements
 
-1. [VirtualBox](https://www.virtualbox.org/)
-2. [Vagrant](http://www.vagrantup.com/) 1.8.5+
-3. [git](https://git-scm.com/)
-4. [ansible](https://www.ansible.com/community) 2.3+
+1. [VirtualBox](https://www.virtualbox.org/wiki/Downloads) 5.1+
+2. [Vagrant](https://www.vagrantup.com/downloads.html) 1.95+
```

Comment:
Assuming that 1.95+ here is supposed to be 1.9.5+.

Reply:
@MarcusBarnes Good catch! that should be 1.9.5+ for sure. Thanks!